src/store/orders/ordersSlice.tsx

```diff
@@ -77,12 +116,99 @@ export const createOrder = createAsyncThunk<Order, void, { state: RootState }>(
       topping: currentTopping,
       name: currentName.length > 0 ? currentName : currentDefaultName,
       addToOrder: currentAdds,
-      date: 0,
-      id: new Date().getTime(),
+      orderId: new Date().getTime(),
     };
   }
 );
 
+export const createRandomOrder = createAsyncThunk<
+  Order,
+  void,
+  { state: RootState }
+>("orders/randomOrder", (_, thunkAPI) => {
+  const state = thunkAPI.getState() as RootState;
+  const breadVariants = breadVariantsSelector(state);
+  const cheeseVariants = cheeseVariantsSelector(state);
+  const meatVariants = meatVariantsSelector(state);
+  const dressingVariants = dressingVariantsSelector(state);
+  const vegetablesVariants = vegetableVariantsSelector(state);
+  const eggVariants = eggVariantsSelector(state);
+  const spreadVariants = spreadVariantsSelector(state);
+  const servingVariants = servingVariantsSelector(state);
+  const toppingVariants = toppingVariantsSelector(state);
+  const currentName = nameSelector(state);
+  const currentDefaultName = defaultNameSelector(state);
+
+  const randomizeValue = (inputArr: string[]): [string] => {
```

Comment:
i would move all these generic helper functions to some separate folder , for example "src/utils"  to clean it up a little bit

Reply:
It's done. I hope you mean something like this.

Reply:
moving away from here ingredientsVariants is fine as well :) but meant more this randomizeValue , randomizeArray functions, they are quite generic, theoretically, you could take them and copy-paste to some other project , this usualy means that it might be a good idea to move them to seperate files/folders